my_blink.ino

```diff
@@ -25,7 +25,7 @@ void setup() {
  */
 void loop() {
     digitalWrite(led, HIGH);   /* turn the LED on (HIGH is the voltage level */
-    delay(1000);               /* wait for a second */
+    delay(3000);               /* wait for a second */
     digitalWrite(led, LOW);    /* turn the LED off by making the voltage LOW */
-    delay(1000);               /* wait for a second */
+    delay(3000);               /* wait for a second */
```

Comment:
I believe it would be better if the light was off for 4 seconds and on for just 3.